pkg/metrics/server/setup.go

```diff
@@ -30,6 +30,7 @@ import (
 
 const (
 	PortDefault = 2112
+	IPDefault   = "127.0.0.1"
```

Comment:
let's use "0.0.0.0" for backward compatibility

Reply:
Ack, will update.